src/main.rs

```diff
@@ -430,6 +447,58 @@ INCLUDE device.x
         );
     }
 
+    #[test]
+    fn parse_plus_origin_k() {
```

Comment:
Great that you are adding tests! Could you add another one without `K` or `M`? E.g. `ORIGIN = 0x20020000 + 1000`

Reply:
Can you please add some tests for `fn perform_addition` as well?